lib/matplotlib/tests/test_ticker.py

```diff
@@ -192,6 +192,69 @@ def test_additional(self, lim, ref):
 
         assert_almost_equal(ax.yaxis.get_ticklocs(minor=True), ref)
 
+    @pytest.mark.parametrize(
+        'lim, ref, use_rcparam', [
```

Comment:
You should stack the `use_rcparam` parametrization so you don't have to write out `lim, ref` values twice.
```suggestion
    @pytest.mark.parametrize('use_rcparam', [False, True])
    @pytest.mark.parametrize(
        'lim, ref', [
```